### DISCUSSION TITLE
onedrive.live.com: breakage

### DISCUSSION THREAD

**Discussion by stultitia** [uBlockOrigin/uAssets#21772] [Q&A]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://onedrive.live.com/`

### Category

breakage

### Description

This is my first time giving a feedback on the extension. And actually, the issue is more in not understanding why some scripts are blocked when given explicit exceptions. Please see attached screenshots. I've tried removing the 3rd party script and frame blocks, as well, and obtained the same result. This is while working on a OneNote notebook and the website appears to work properly as of now. Please excuse me if I have used a wrong medium to report such a matter. I am open to a temporary block if I misunderstood the medium and its use.

Additonal note: I just found out there is a [bug section](https://github.com/uBlockOrigin/uBlock-issues), should I have preferred there?

### Other extensions used

- Tab Stash
- Multi-Containers
- Firefox Notes
- Private Bookmarks
- Privacy Badger (not enabled on reported site)

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Screen Shot 2024-01-04 at 17 14 25](https://github.com/uBlockOrigin/uAssets/assets/12058403/07d2e803-2afe-4427-85bc-6f38e2a95171)
<img width="634" alt="Screen Shot 2024-01-04 at 17 38 24" src="https://github.com/uBlockOrigin/uAssets/assets/12058403/c7149617-2b68-4ee8-aad3-1581951fc664">
<img width="633" alt="Screen Shot 2024-01-04 at 17 38 29" src="https://github.com/uBlockOrigin/uAssets/assets/12058403/ac2f903c-5c00-4b6c-be42-fdef776a335c">

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.54.0
Firefox: 115
filterset (summary):
 network: 104784
 cosmetic: 44202
 scriptlet: 18536
 html: 1991
listset (total-discarded, last-updated):
 added:
  fanboy-thirdparty_social: 68-0, now
 default:
  user-filters: 2-0, never
  ublock-filters: 36987-139, 57m
  ublock-badware: 7744-11, 57m
  ublock-privacy: 946-0, 57m
  ublock-unbreak: 2222-1, 57m
  easylist: 76715-610, 57m
  easyprivacy: 33103-793, 57m
  plowe-0: 3788-1, now
  urlhaus-1: 9530-0, now
  ublock-quick-fixes: 174-64, 57m
filterset (user): [array of 3 redacted]
trustedset:
 added: [array of 131 redacted]
 removed:
  about-scheme
  edge-scheme
switchRuleset:
 added: [array of 1 redacted]
hostRuleset:
 added: [array of 5158 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 13596 ms (selfie)
 maxAssetCacheWait: 529 ms
popupPanel:
 blocked: 2
 network:
  live.com: 1
  office.com: 1
```
</details>

**Comment by big-cat-loaf** [comment]
I'm not an expert, contributor, or staff member, but I know a bit uBO
First, when you want to ask a question, you can ask it at https://www.reddit.com/r/uBlockOrigin/.
Second, if you want to learn more about uBO, you can go to https://github.com/gorhill/uBlock/wiki.
Third To write filters, the links could help.
Basic form Adblocker Plus: https://help.adblockplus.org/hc/en-us/articles/360062733293-How-to-Write-Filters
Adcanced from the uBO wiki: https://github.com/gorhill/uBlock/wiki/Static-filter-syntax
